packages/web/pages/index.tsx

```diff
@@ -94,6 +96,12 @@ const Landing: React.FC = () => {
 
   return (
     <>
+      <HeadComponent
+        title="MetaGame"
+        description="To find your metagame means to play life in the optimal way. Coordinating with others on building a better world; doing things that create a a positive impact make you happy AND earn you money."
+        url="https://metagame.wtf"
+        img="https://metagame.wtf/_next/image?url=%2Fassets%2Fmanifesto.jpg&amp;w=1080&amp;q=75"
```

Comment:
How sure are we that this will work ? 
IMO it would be much cleaner to move the manifesto.jpg to the public folder and make it simply 
`https://metagame.wtf/manifesto.jpg` ?

Reply:
> How sure are we that this will work ? IMO it would be much cleaner to move the manifesto.jpg to the public folder and make it simply `https://metagame.wtf/manifesto.jpg` ?

I assumed it would as there are other instances of that component using the same format but happy to change it if you think would be better. Is there a way to do it without hard-coding the hostname?